src/Anemone/Embed.hs

```diff
@@ -1,19 +1,23 @@
 {-# LANGUAGE NoImplicitPrelude #-}
 {-# LANGUAGE TemplateHaskell #-}
-module Anemone.Embed
-  ( anemone_base_h
+module Anemone.Embed (
+    anemone_base_h
+  , anemone_grisu2_h
   , anemone_mempool_h
-  , anemone_mempool_c
   ) where
 
-import qualified Data.FileEmbed as FileEmbed
 import           Data.ByteString (ByteString)
+import qualified Data.FileEmbed as FileEmbed
+
 
 anemone_base_h :: ByteString
-anemone_base_h = $(FileEmbed.embedFile "csrc/anemone_base.h")
+anemone_base_h =
+  $(FileEmbed.embedFile "csrc/anemone_base.h")
 
-anemone_mempool_h :: ByteString
-anemone_mempool_h = $(FileEmbed.embedFile "csrc/anemone_mempool.h")
+anemone_grisu2_h :: ByteString
+anemone_grisu2_h =
+  $(FileEmbed.embedFile "csrc/anemone_grisu2.h")
 
-anemone_mempool_c :: ByteString
-anemone_mempool_c = $(FileEmbed.embedFile "csrc/anemone_mempool.c")
```

Comment:
I figured it didn't make sense to expose the implementation, `anemone_mempool.c`, we might as well link to it using `rdynamic` like we do with `zebra` and `piano`.